tests/Microsoft.DotNet.Docker.Tests/MonitorImageTests.cs

```diff
@@ -118,6 +118,11 @@ public void VerifyEnvironmentVariables(MonitorImageData imageData)
             // DefaultProcess filter should select a process with a process ID of 1
             variables.Add(new EnvironmentVariableInfo("DefaultProcess__Filters__0__Key", "ProcessId"));
             variables.Add(new EnvironmentVariableInfo("DefaultProcess__Filters__0__Value", "1"));
+            // Existing (orphaned) diagnostic port should be delete before starting server
+            if (imageData.Version >= ImageVersion.V6_2)
```

Comment:
I'd prefer:

```suggestion
            if (imageData.Version > ImageVersion.V6_1)
```

That way, when we eventually delete `V6_1`, the compiler will let us know about this and we can determine that there's no need for the conditional.